src/atoms/Pun/Pun.test.js

```diff
@@ -60,9 +61,30 @@ describe("<Pun />", () => {
       expect(messageCell).toBeInTheDocument();
       expect(row).toContainElement(messageCell);
 
-      const likeButton = screen.queryAllByRole("button", { name: /like/i, container: row });
+      const likeButton = screen.queryAllByRole("button", {
+        name: /like/i,
+        container: row,
+      });
       expect(likeButton.length).toBeGreaterThan(0);
       expect(likeButton[0]).toBeInTheDocument();
     }
   });
+
+  it("Render Pun with correct class", () => {
```

Comment:
I don't know if testing the presence of a class is the best option, since the idea is to test the user experience. What do you think?

Reply:
Hmm, let me try to convince you, "vender meu peixe".
My style is rendered by the css that is linked in the className, so if for some reason i do not have the correct class, my component will not have the correct style and it will not be a good user experience, cuz everything is going to be  a mess.